ports/espressif/boards/bpi_picow_s3/mpconfigboard.mk

```diff
@@ -0,0 +1,22 @@
+USB_VID = 0x303A
+USB_PID = 0x812C
+USB_PRODUCT = "BPI-PicoW-S3"
+USB_MANUFACTURER = "BananaPi"
+
+IDF_TARGET = esp32s3
+
+INTERNAL_FLASH_FILESYSTEM = 1
+LONGINT_IMPL = MPZ
+
+# The default queue depth of 16 overflows on release builds,
+# so increase it to 32.
+CFLAGS += -DCFG_TUD_TASK_QUEUE_SZ=32
+
+CIRCUITPY_ESP_FLASH_MODE = dio
+CIRCUITPY_ESP_FLASH_FREQ = 80m
+CIRCUITPY_ESP_FLASH_SIZE = 8MB
+
+#CIRCUITPY_BITBANG_NEOPIXEL = 1
+
+# Include these Python libraries in firmware.
+FROZEN_MPY_DIRS += $(TOP)/frozen/Adafruit_CircuitPython_NeoPixel
```

Comment:
Are you freezing this for convenience? Usually we don't freeze libraries unless there are space reasons.

Reply:
Yes, this is very convenient indeed.
Can I be allowed to freeze it?

Reply:
It's your board, so sure.